docs/api/enums.rst

```diff
@@ -2369,6 +2372,14 @@ of :class:`enum.Enum`.
 
     .. versionadded:: 2.5
 
+    .. attribute:: durable
+
+        Represents a durable one-time purchase.
+
+    .. attribute:: consumable
+
+        Represents a consumable one-time purchase.
+
```

Comment:
Maybe we can define "durable" and "consumable"

Reply:
Good idea, I'll work on that later today

Reply:
Suggestions made